tests/phpunit.xml.bak

```diff
@@ -0,0 +1,24 @@
+<phpunit
```

Comment:
this looks like is was accidentally committed

Reply:
right, removed this file again.